app/_src/production/cp-deployment/multi-zone.md

```diff
@@ -66,6 +82,81 @@ The global control plane on Kubernetes must reside on its own Kubernetes cluster
     By default, it's exposed on {% if_version lte:2.1.x %}[port 5685](/docs/{{ page.version }}/networking/networking){% endif_version %}{% if_version gte:2.2.x %}[port 5685](/docs/{{ page.version }}/production/deployment/networking/){% endif_version %}. In this example the value is `35.226.196.103:5685`. You pass this as the value of `<global-kds-address>` when you set up the zone control planes.
 
 {% endtab %}
+
+{% if_version gte:2.2.x %}
+{% tab global-control-plane Universal on Kubernetes using Helm %}
+
+1. Set `controlPlane.environment=universal` and `controlPlane.mode=global` in the chart (`values.yaml`).
+
+1. Set `controlPlane.secrets` with database sensitive information
```

Comment:
shouldn't this be `2.`?

Reply:
It auto increments:

![image](https://user-images.githubusercontent.com/2753650/230077587-7c7bf768-a34b-4cfe-babd-9d2aee810044.png)

that way I don't have to fix it manually.

Reply:
If you want me to I can change this to the correct numeration :D